src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseGroupAuditService.java

```diff
@@ -23,7 +22,7 @@ public void updateAuditTable(final CaseGroup caseGroup, final UUID partyId) {
     auditEntity.setCaseGroupFK(caseGroup.getCaseGroupPK());
     auditEntity.setStatus(caseGroup.getStatus());
     auditEntity.setPartyId(partyId);
-    auditEntity.setCreatedDateTime(DateTimeUtil.nowUTC());
+    auditEntity.setCreatedDateTime(caseGroup.getChangeStateTimestamp());
```

Comment:
Not convinced we use this table for anything, but right to update

Reply:
I don't believe it is but I thought for the sake on continuity to update it.